Skype/SDN/articles/echopercentmicin-element-qualitypropertiestype-complextype-skype-sdn-2-2-d.md

```diff
@@ -1,6 +1,7 @@
 ﻿---
-title: EchoPercentMicIn element (QualityPropertiesType complexType) 
+title: EchoPercentMicIn element, QualityPropertiesType complexType
 TOCTitle: EchoPercentMicIn element
+description: Information about the EchoPercentMicIn element.
```

Comment:
word count falls below the threshold, but feel free to take the comment with a grain of salt. :)